src/com/github/Kraken3/AFKPGC/Warning.java

```diff
@@ -0,0 +1,18 @@
+package com.github.Kraken3.AFKPGC;
+
+/**
+ * Lightweight warning message concerning kick nearness.  
+ */
+class Warning{

```

Comment:
Didn't even check. If you want to make a pull against my repo pulling out this dead code and the dead config options, that'd be great.